CMSgov/design-system
# [WNMGDS-566] Always compile to ESM and Common JS

### Summary
- Based off [this conversation](https://adhoc.slack.com/archives/C645GLLNN/p1595877042204600?thread_ts=1595536700.177800&cid=C645GLLNN), we decided it would be best for child DS to also be dual packages, and compile to both CommonJS and ESM

### How to test
- Go to either of the child ds PRs and use this local branch as the `@cmsgov/design-system-scripts` dependency
- Update the `babel.config.json` to remove the `modules` property
- Run the build scripts and ensure both ESM and CommonJS are outputted
